apps/api/src/app/auth/framework/auth.guard.ts

```diff
@@ -29,7 +29,7 @@ export class JwtAuthGuard extends AuthGuard('jwt') {
       return this.authService.apiKeyAuthenticate(key).then<any>((result) => {
         request.headers.authorization = `Bearer ${result}`;
 
-        return super.canActivate(context);
+        return true;
```

Comment:
No need to validate again since we already validated the token above

Reply:
Can you elaborate where exactly? I think I am missing it.

Reply:
in the `apiKeyAuthenticate` function